src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -47,17 +56,28 @@ public ResponseEntity<Object> createNewSubmission(
             @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId,
             @RequestHeader(value = ERIC_IDENTITY) String userId,
             HttpServletRequest request) {
-        String passthroughTokenHeader = request.getHeader(ApiSdkManager.getEricPassthroughTokenHeader());
 
         var logMap = new HashMap<String, Object>();
         logMap.put(TRANSACTION_ID_KEY, transaction.getId());
-        ApiLogger.infoContext(requestId, "Calling service to create Overseas Entity Submission", logMap);
 
         try {
+            if(isValidationEnabled) {
+                var validationErrors = overseasEntitySubmissionDtoValidator.validate(overseasEntitySubmissionDto, new Errors(), requestId);
+
+                if (validationErrors.hasErrors()) {
+                    ApiLogger.infoContext(requestId, "Validation errors : " + validationErrors);
```

Comment:
Could potentially log this as a warning (or even error) since, at the moment, any requests getting through with validation errors should have already been caught at the web layer and shouldn't have got this far.

Reply:
Log as an error added new exception as argument